src/main/java/frc/robot/drivetrain/Drivetrain.java

```diff
@@ -72,18 +72,54 @@ public void initDefaultCommand() {
     // setDefaultCommand(new SampleDrive());
   }
 
-  public void setRawPercentOutput(double leftPercent, double rightPercent) {
+  // private void setRawPercentOutput(double leftPercent, double rightPercent) {
+  //   left.set(ControlMode.PercentOutput, leftPercent);
+  //   right.set(ControlMode.PercentOutput, rightPercent);
+  // }
+
+  // private void setPercentOutput(double leftPercent, double rightPercent) {
+  //   setVelocityOutput(leftPercent * MAX_VELOCITY_IN_FPS, rightPercent * MAX_VELOCITY_IN_FPS);
+  // }
+
+  private void setVelocityOutput(double leftVelocity, double rightVelocity) {
+    left.set(Velocity, leftVelocity);
+    right.set(Velocity, rightVelocity);
+  }
+
+  private void setPowerOutput(double leftPercent, double rightPercent) {
     left.set(ControlMode.PercentOutput, leftPercent);
     right.set(ControlMode.PercentOutput, rightPercent);
   }
 
-  public void setPercentOutput(double leftPercent, double rightPercent) {
-    setVelocityOutput(leftPercent * MAX_VELOCITY_IN_FPS, rightPercent * MAX_VELOCITY_IN_FPS);
+  public enum CommandType {
```

Comment:
Enums should typically be declared at the top of the class

Reply:
Say between the singleton Drivetrain stuff and the declaring of the other constants (wheel diameter, etc.)?